process/transaction/shardProcess.go

```diff
@@ -649,18 +635,19 @@ func (txProc *txProcessor) processUserTx(
 			txHash,
 			err.Error())
 	}
-
+	// TODO: check if there is no problem with hashing a new SCR and not the previous transaction. Are the hashes equal?
```

Comment:
the SCR is where the execution continues.

Reply:
ok